tikv-configuration-file.md

```diff
@@ -400,6 +400,7 @@ Configuration items related to storage.
         + When API V2 is used, you are expected to set `storage.enable-ttl = true` at the same time. Because API V2 supports the TTL feature, you must turn on `enable-ttl` explicitly. Otherwise, it will be in conflict because `storage.enable-ttl` defaults to `false`.
         + When API V2 is enabled, you need to deploy at least one tidb-server instance to reclaim expired data. Note that this tidb-server instance cannot provide read or write services. To ensure high availability, you can deploy multiple tidb-server instances.
         + Client support is required for API V2. For details, see the corresponding instruction of the client for the API V2.
+        + From v6.2.0, RawKV Change Data Capture (CDC) can be achieved with the new component [TiKV-CDC](https://github.com/tikv/migration/tree/main/cdc).
```

Comment:
```suggestion
        + Since v6.2.0, you can use change data capture (CDC) with RawKV using the component [TiKV-CDC](https://github.com/tikv/migration/tree/main/cdc).
```